### PR TITLE
Analytics and SEO update

### PR DESCRIPTION
**Pull request by whallin**
### Questions
- [x] Did you preview on your local machine beforehand?
- [x] Did you follow our [commit handbook](https://github.com/Atlas-OS/docs/blob/master/.github/CONTRIBUTING.md#commit-message)?

### Describe your pull request

* Corrects tiny mistakes and improves clarity
* Implements Microsoft Clarity for UX reports
* ~~Uses ``timeago`` rather than ``datetime`` for git-revision-date-localized~~
   * See below.
* ~~Disables creation date lookup (fetch depth can't be set @ Cloudflare Pages)~~
   * Disables git-revision-date-localized completely as the fetch depth is too low during build at Cloudflare Pages, making the creation and last updated date both show the build date and time at Cloudflare Pages.
* Added descriptions and icons to each page
* Removed unneeded index pages
* Updated font for metadata image generation

### CODE REVIEW

**Review by whallin — COMMENTED** [review]
If anyone knows or can figure out how to set the fetch depth to 0 during build at Cloudflare Pages, let us know.

Read [this](https://git-scm.com/docs/git-fetch) for context and see the ``--depth`` flag.

**Comment by JayXTQ** [comment]
> If anyone knows or can figure out how to set the fetch depth to 0 during build at Cloudflare Pages, let us know.
> 
> Read [this](https://git-scm.com/docs/git-fetch) for context and see the `--depth` flag.

I didn't know it is possible to edit the Git command?

**Comment by whallin** [comment]
> I didn't know it is possible to edit the Git command?

@JayXTQ editing the Git command itself will most likely not be possible, yes. However, when using Actions there's [``checkout``](https://github.com/actions/checkout) where you can set the fetch depth using the ``fetch-depth`` property.

Maybe Cloudflare Pages has a setting for it somewhere? Maybe they call it something else?

**Comment by JayXTQ** [comment]
I believe CF Pages has something internal for how they do actions, unless I legit can't see the actions page on my account with my permissions.

**Comment by PencilNavigator** [comment]
maybe we can ask here? https://discord.gg/cloudflaredev

**Comment by JayXTQ** [comment]
Not everything needs to be asked in a Discord server, let's see what can be done first because we don't wanna ask a dumb question.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Welcome to the AtlasOS docs contributing guide
+# Welcome to the AtlasOS documentation contributing guide
```
Comment (he3als) [inline .github/CONTRIBUTING.md:1]: Maybe the contribution guide here could be stripped down (with only repo-specific stuff) after https://github.com/Atlas-OS/docs/pull/76 is merged?

File: overrides/partials/integrations/analytics/custom.html [hunk 2]
```diff
@@ -1 +1,10 @@
-<script defer data-domain="docs.atlasos.net" src="https://insights.tweakcentral.net/js/script.outbound-links.file-downloads.js"></script>
\ No newline at end of file
+<!-- Plausible -->
+<script defer data-domain="docs.atlasos.net" src="https://insights.tweakcentral.net/js/script.outbound-links.file-downloads.js"></script>
+<!-- Microsoft Clarity -->
```
Comment (he3als) [inline overrides/partials/integrations/analytics/custom.html:3]: How privacy impactful is this?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (whallin) [reply .github/CONTRIBUTING.md:1]: I think @JayXTQ or @PencilNavigator would be the best to answer anything regarding the contribution guide within this repository for the documentation.

However, I agree that this repository's contribution guide should contain more information about how to set up a development environment and so forth. Just like #76 does for the Atlas playbook.

Maybe leave this question for an internal discussion for another PR where that might occur?6

Comment (JayXTQ) [reply .github/CONTRIBUTING.md:1]: I made that guide before we even did the internal chat, we can modify it how we wish.

File: overrides/partials/integrations/analytics/custom.html (hunk 2 above)
Comment (whallin) [reply overrides/partials/integrations/analytics/custom.html:3]: [Plausible](https://plausible.io/) is privacy respecting as it doesn't track users between sessions. Clarity claims to not track users between sessions unless we collect consent, which we don't, as we don't want that cross-session tracking.

Comment (he3als) [reply overrides/partials/integrations/analytics/custom.html:3]: Ah okay, that's fine then, as long as it isn't somehow tracking users in an invasive way.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (whallin) [reply .github/CONTRIBUTING.md:1]: @he3als what suggestions do you have then for change? Maybe you'd like to turn it into an internal discussion with @JayXTQ for now? I feel like that change is quite unfitting for this PR.

Comment (he3als) [reply .github/CONTRIBUTING.md:1]: I agree, for now, no changes. Once this is merged, I might include some changes in https://github.com/Atlas-OS/docs/pull/76 and see what you think about it.

**Review by he3als — APPROVED** [review]

**Review by Xyueta — APPROVED** [review]

**Pull request merged**